.whatsnew.json

```diff
@@ -0,0 +1,54 @@
+{
+    "$schema": "https://raw.githubusercontent.com/dotnet/docs-tools/main/WhatsNew.Infrastructure/Configuration/reposettings.schema.json",
+    "docSetProductName": ".NET Aspire",
+    "rootDirectory": "docs/",
+    "docLinkSettings": {
+        "linkFormat": "relative",
+        "relativeLinkPrefix": "../"
+    },
+    "navigationOptions": {
+        "maximumNumberOfArticles": 3,
+        "tocParentNode": "Latest documentation updates",
+        "repoTocFolder": "docs/whats-new",
+        "indexParentNode": "Latest documentation updates",
+        "repoIndexFolder": "docs/whats-new"
+    },
+    "areas": [
+        {
+            "names": [ "fundamentals" ],
+            "heading": ".NET Aspire fundamentals"
```

Comment:
You might not want ".NET Aspire" in every heading. Every heading is for .NET Aspire, so it seems redundant.